SofaKernel/modules/Sofa.Config/src/sofa/config.h.in

```diff
@@ -305,6 +305,11 @@ class DeprecatedAndRemoved {};
         "v21.12 (PR#2292)", "v22.06", \
         "Use the Messaging API instead of using SofaOStream and sout/serr/sendl.")
 
+#define SOFA_ATTRIBUTE_DEPRECATED__OBJECTFACTORY(msg) \
+    SOFA_ATTRIBUTE_DEPRECATED( \
+        "v21.12 (PR#xxx)", "v22.12", \
+        "A new API has been proposed for ObjectFactory. Please update your code to use this one." msg)
+
```

Comment:
Should be moved to SofaKernel/modules/SofaCore/src/sofa/core/config.h.in

Reply:
Right.

NB: if this is the guideline we probably should we move all the deprecation that are in Sofa.Config/config.h.in into their corresponding package  ?

Reply:
Yes we should :+1: